src/tx.rs

```diff
@@ -115,7 +115,7 @@ pub struct StaticTx(RwLock<Option<Sender<Payload>>>);
 
 impl StaticTx {
     const fn none() -> Self {
-        StaticTx(RwLock::const_new(None))
+        StaticTx(RwLock::const_new(parking_lot::RawRwLock::INIT, None))
```

Comment:
You should use `parking_lot::const_rwlock(None)`.

Reply:
Fixed in commit b208d2a